### ISSUE TITLE
leia 18.7 kodi 

### ISSUE THREAD

**Issue by shyguy23** [Tempest0580/tempest#109]
just upgraded to leia 18.7 and when attempting to install Tempest addon  i get error xmbc python 3.00  dependacy not met

**Comment by eeden003** [comment]
This is correct ! Tempest is "dead"for months and months, no updates.  And....the newest Kodi is 19.4 ! use thát Kodi version please 
!
error xmbc python 3.00 dependacy not met: means that you NEED Kodi19.X  !
